test/math.wsky

```diff
@@ -127,25 +147,85 @@ var testAbs = {
     nearEqual(0.0, abs(0.0));
 };
 
+var testExp = {
+  var exp = math.exp;
+
+  nearEqual(1.0, exp(0));
+  nearEqual(math.E, exp(1));
+  nearEqual(7.38905, exp(2));
+  nearEqual(0.135335, exp(-2));
+};
+
+var testLogarithme = {
+  var log = math.log;
+  var log2 = math.log2;
+  var log10 = math.log10;
+  var log1p = math.log1p;
+
+  nearEqual(0.0, log(1));
+  nearEqual(123.0, log(math.exp(123)));
+  nearEqual(-100.0, log(math.exp(-100)));
+
+  nearEqual(8.0, log2(256));
+  nearEqual(1.0, log2(2));
+
+  nearEqual(1.0, log10(10));
+  nearEqual(10.0, log10(10000000000));
+
+  equal('9.99999995e-09', log1p(0.00000001).toString);
```

Comment:
Didn't find the way to write 9.99999995 * 10^-9 in a proper way :s 
And i couldn't use nearEqual(0) or it will not test the special properties of log1p ^^